src/styles/blocks/bike.scss

```diff
@@ -0,0 +1,35 @@
+.bike {
+  &__photo-container {
+    margin-bottom: 32px;
+    z-index: 0;
+    transition: transform 0.3s, z-index 0.3s;
+
+    &:hover {
+      transform: scale(105%);
+      z-index: 1;
+    }
+  }
```

Comment:
z-index does not work without a position property different from the default and I don't know for what you need it at all, better remove